app/controllers/chambers/documents_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Chambers::DocumentsController < ApplicationController
+  before_action :authenticate_user!
+  def create
+    document = Document.new(document_params)
+    if document.save
+      flash[:notice] = "Document Upload Successful"
+      candidate = Candidate.find(params['document']['candidate_id'])
+      redirect_to chambers_candidate_path(candidate.order,candidate)
+    else
+      flash[:alert] = "Document Upload Unsuccessful: #{document.errors.full_messages.to_sentence}"
+    end
+  end
+  private
+  def document_params
+    peerage = Candidate.find(params['document']['candidate_id']).peerage_type 
+    peer = current_user.peer(peerage)
```

Comment:
Would `peer = current_user.peer(params[:peerage])` work instead?

Reply:
ah, I see - I hadn't seen that images/documents wasn't within the `scope '/:peerage'` but I guess that makes sense.